diesel_derives/tests/auto_type.rs

```diff
@@ -52,7 +52,8 @@ table! {
         multirange -> Multirange<Integer>,
         timestamptz -> Timestamptz,
         name -> Text,
-        text_array -> Array<Text>
+        text_array -> Array<Text>,
+        record -> Record,
```

Comment:
We would need to define the record type here as `Record<(Integer, Text)>` or something like that to fix the compiler error.

Reply:
Sure, I'll make the changes